styx-common/src/main/java/com/spotify/styx/serialization/PersistentEvent.java

```diff
@@ -322,6 +322,25 @@ public Event toEvent() {
     }
   }
 
+  public static class Dequeue extends PersistentEvent {
+
+    public final Set<String> resourceIds;
+
+    @JsonCreator
+    public Dequeue(
+        @JsonProperty("workflow_instance") String workflowInstance,
+        @JsonProperty("resource_ids") Optional<Set<String>> resourceIds) {
```

Comment:
This was also consistent with `Submit` handling below but b5777c0 made it inconsistent...